Amplify/Categories/API/ClientBehavior/AmplifyAPICategory+ReachabilityBehavior.swift

```diff
@@ -0,0 +1,15 @@
+//
+// Copyright 2018-2019 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import Combine
+extension AmplifyAPICategory: APICategoryReachabilityBehavior {
+    @available(iOS 13.0, *)
+    public func reachability(apiName: String?) throws -> AnyPublisher<ReachabilityUpdate, Never>? {
```

Comment:
As noted elsewhere, default this to `nil` so callers can just invoke `Amplify.API.reachability()`

Reply:
sync'ed offline. updated & resolving.